### ISSUE TITLE
PI in math package is 0.0

### ISSUE THREAD

**Issue by dram** [StanzaOrg/lbstanza-old#22]
When compile and run following code:

```
defpackage tests:
  import core
  import math

println(PI)
```

Following result seems not to be correct:

```
0.000000000000000
```

**Comment by CuppoJava** [comment]
Can you verify that you're using 0.9.2? Version 0.9.1 introduced some floating point bugs.

**Comment by dram** [comment]
I tried both 0.9.1 and 0.9.2, results are same.

**Comment by CuppoJava** [comment]
Mmm you're right. We used the wrong bootstrap compiler to build 0.9.2. A fix is incoming.

**Comment by CuppoJava** [comment]
Fixed in 0.9.3.
